packages/tfjs-node-helpers/src/testing/metrics/f1-score.ts

```diff
@@ -1,16 +1,7 @@
-import { ConfusionMatrix } from '../confusion-matrix';
-import { Metric } from '../metric';
-import { MetricCalculator } from '../metric-calculator';
+import { FBetaScoreMetricCalculator } from './fbeta-score';
 
-export class F1ScoreMetricCalculator extends MetricCalculator {
-  public calculate(trueValues: Float32Array, predictedValues: Float32Array): Metric {
-    const { tp, fp, fn } = new ConfusionMatrix(trueValues, predictedValues);
-    const precision = tp / (tp + fp);
-    const recall = tp / (tp + fn);
-
-    return new Metric({
-      title: 'F1Score',
-      value: (2 * precision * recall) / (precision + recall)
-    });
+export class F1ScoreMetricCalculator extends FBetaScoreMetricCalculator {
```

Comment:
@VladimirGlinskii I think we can remove `F1ScoreMetricCalculator`, since we have `FBetaScoreMetricCalculator` and can pass it `1` if needed.